### ISSUE TITLE
Investigate uploading new images from Whitehall to Asset Manager

### ISSUE THREAD

**Issue by chrisroos** [alphagov/asset-manager#185]
This is a bit of a placeholder to prompt us to investigate what's required to change the Whitehall app to upload new images to Asset Manager. We imagine updating this issue and possibly creating new issues as the work required becomes clearer.

**Comment by floehopper** [comment]
This spike is relevant to this issue: https://github.com/alphagov/whitehall/compare/spike-use-asset-manager-for-attachments

**Comment by floehopper** [comment]
Yesterday @chrisroos & I built on top of the spike from #186 in relation to this issue in the following branches:

* https://github.com/alphagov/asset-manager/tree/serve-whitehall-images-from-asset-manager
* https://github.com/alphagov/govuk-puppet/tree/serve-whitehall-images-from-asset-manager
* https://github.com/alphagov/whitehall/tree/serve-whitehall-images-from-asset-manager
* https://github.com/alphagov/gds-api-adapters/tree/serve-whitehall-images-from-asset-manager

**Comment by floehopper** [comment]
One of the issues we ran into yesterday was how to serve up the Whitehall `thumbnail-placeholder.png` image after we've moved Whitehall images to be served from Asset Manager. In the end we added it to Asset Manager as a Rails asset, although there are a few problems with doing that.

I just had a look at the static app and noticed it serves up some [generic images](https://github.com/alphagov/static/tree/master/app/assets/images) and [some content-specific images](https://github.com/alphagov/static/tree/master/app/assets/content-images). I wonder whether that might be a better place.
